src/components/bottomDrawer/stage/stage.js

```diff
@@ -29,13 +29,15 @@ class Stage extends Component {
               textColor="primary"
               scrollButtons="on"
               scrollable>
-          <Tab label="Friday - July 13th" />
+          <Tab label="Friday July 12th" />
           <Tab label="Saturday - July 14th" />
           <Tab label="Sunday - July 15th" />
         </Tabs>
 
         {value === 0 && <TabContainer>
-          (Beer + Cocktail Garden opens and only Food Trucks available 5 PM-10 PM)<br /><br />
+          Friday evening from 5:00 p.m. - 11:00 p.m. we kick off the festival with a Ballard Block Party featuring a DJ, #All4Doras and NITE WAVE.<br/>
+          The Beer Garden will be open and five Food Trucks are available for your dining pleasure!<br/>
```

Comment:
There are six food trucks listed in the map PDF. Need to confirm with Devin and Anndrea.